.gitignore

```diff
@@ -1,21 +1,14 @@
-# Build files
-pom.xml
-*jar
-/lib/
-/classes/
+
+# Leiningen
 /target/
-/docs/
-/pkg/
-/log/
-*.tar.gz
-version
-junit
-
-# Emacs files
+
+# Emacs
 *#
+*~
+.#*
 
 # Local overide for testing config
-test-resources/config/local.clj
+/test-resources/config/local.clj
 
 # Vagrant cache folder & vagrantfile if used
```

Comment:
I also wondered if we really don't want git status to tell us when there are 5 vagrant dirs in say src/puppetlabs/puppetdb...